assets/src/components/map/markers/stopMarker.tsx

```diff
@@ -0,0 +1,304 @@
+            fill="#3c3f4c"
+          />
+        </svg>
+      )}
+      {type === StopType.Large && (
+        <svg className={embeddedSvgClasses} viewBox="0 0 20 20" fill="none">
+          <FocusCircle cx="10" cy="10" r={8 + strokeWidthOffset} />
+          <circle
+            className={joinClasses(circleClasses)}
+            cx="10"
+            cy="10"
+            r="8"
+            fill="white"
+            stroke="#3C3F4C"
+            strokeWidth="1"
+          />
+          <path
+            className={busClasses}
+            fillRule="evenodd"
+            clipRule="evenodd"
+            d="m 15,9.4348766 -0.4861,-3.62308 c -0.1268,-0.65342 -0.5609,-0.9136 -1.2041,-1.18057 -1.0697,-0.35849 -2.1864,-0.55002 -3.314,-0.57206 -1.12426,0.02204 -2.23845,0.21442 -3.30478,0.57206 -0.63644,0.26697 -1.07809,0.52715 -1.21243,1.18057 L 5,9.4348766 v 5.0044904 h 0.89337 v 0.8534 c 0.00587,0.2255 0.13182,0.4306 0.32913,0.5348 0.04534,0.022 0.08984,0.0449 0.12679,0.0593 0.03442,0.0128 0.06969,0.0229 0.10495,0.0297 0.0571,0.0153 0.11419,0.0229 0.17212,0.0229 h 0.25441 c 0.37952,0.0237 0.70949,-0.2636 0.74056,-0.6467 v -0.8534 h 4.82957 v 0.8534 c 0.0059,0.2255 0.131,0.4306 0.3283,0.5348 0.0453,0.022 0.0907,0.0449 0.1276,0.0593 0.0344,0.0128 0.0689,0.0229 0.105,0.0297 0.0554,0.0153 0.1141,0.0229 0.1721,0.0229 h 0.2544 c 0.3795,0.0237 0.7078,-0.2636 0.7397,-0.6467 v -0.8534 H 15 Z m -7.32822,-4.49177 h 4.70362 c 0.225,0 0.4072,0.18136 0.4114,0.40849 -0.0042,0.22629 -0.1864,0.4085 -0.4114,0.4085 H 7.67178 c -0.22502,0 -0.40806,-0.18221 -0.41226,-0.4085 0.0042,-0.22713 0.18724,-0.40849 0.41226,-0.40849 z m -1.17382,4.4138 0.33333,-2.47725 c 0.03107,-0.17119 0.18304,-0.29238 0.356,-0.28391 H 12.859 c 0.1721,-0.00678 0.3241,0.11357 0.3568,0.28391 l 0.3804,2.47725 v 0.11103 c -0.005,0.19492 -0.1637,0.34832 -0.3568,0.34578 H 6.84137 c -0.19144,-0.00763 -0.34341,-0.16611 -0.34341,-0.36019 -0.00672,-0.03136 -0.00672,-0.06441 0,-0.09662 z m 0.47347,3.8934604 c -0.56759,0.0263 -1.05626,-0.4026 -1.11083,-0.9729 0,-0.2848 0.11167,-0.5594 0.3115,-0.7602 0.19983,-0.2017 0.47103,-0.3153 0.75315,-0.3153 0.58606,-0.0441 1.09907,0.3932 1.15701,0.9831 -0.00672,0.2907 -0.12763,0.5661 -0.33669,0.7653 -0.20823,0.2009 -0.48699,0.3085 -0.77414,0.3 z m 6.00827,0 c -0.5684,0.0263 -1.0562,-0.4026 -1.1108,-0.9729 0,-0.2848 0.1117,-0.5594 0.3123,-0.7602 0.199,-0.2017 0.4694,-0.3153 0.7523,-0.3153 0.5861,-0.0441 1.1,0.3932 1.1562,0.9831 -0.0059,0.2907 -0.1268,0.5661 -0.335,0.7653 -0.2082,0.2009 -0.4878,0.3085 -0.775,0.3 z"
+            fill="#3c3f4c"
+          />
+        </svg>
+      )}
+    </svg>
+  )
+}
+
+interface StopMarkerProps extends Partial<MarkerProps> {
```

Comment:
Oh actually I meant to remove this[^1] as it actually hides that `posistion` is required, thanks for commenting on this!

Responding to the quesiton with my original intent though: I was thinking that there were some marker props that I was going to fill in that the user of this API didn't need to be concerned about and if there were any requirements of the react-leaflet Marker that were required, the user shouldn't get a type error on a prop that's already handled. Given that `posistion` is not handled by this component, `MarkerProps` should not be `Parital`

[^1]: I actually thought I had already removed this, but I must not have committed it or something.

Reply:
Oh actually I do provide `posistion`, it's `Omit<_, "posistion">` that I want, because the user cannot specify that property, but can specify basically everything else.

Reply:
8e7da8ec